caluma/caluma_form/tests/test_document.py

```diff
@@ -1444,3 +1444,77 @@ def test_document_modified_content_properties(
     assert node["modifiedContentAt"] == sub_a.modified_at.isoformat()
     assert node["modifiedContentByUser"] == sub_a.modified_by_user
     assert node["modifiedContentByGroup"] == sub_a.modified_by_group
+
+
+@pytest.mark.parametrize(
+    "question__type,answer__value",
+    [
+        (Question.TYPE_CHOICE, "somevalue"),
+        (Question.TYPE_MULTIPLE_CHOICE, ["somevalue", "anothervalue"]),
+        (Question.TYPE_DYNAMIC_CHOICE, "5.5"),
+        (Question.TYPE_DYNAMIC_MULTIPLE_CHOICE, ["5.5"]),
+        (Question.TYPE_TEXT, "somevalue"),
+    ],
+)
+def test_selected_options(
+    db,
+    snapshot,
+    document,
+    answer,
+    question_option_factory,
+    dynamic_option_factory,
+    schema_executor,
+    question,
+):
+    query = """
+        query AllDocumentsQuery($search: String) {
+          allDocuments(search: $search) {
```

Comment:
the `search` filter is deprecated (should be `filter: [ {"search": $search} ]`, but I think for this test it's not needed at all. You could just query `node($globalid)` instead and go for the answer directly. Would make the test a little bit shorter

Reply:
Done.